src/app/(home)/page.tsx

```diff
@@ -2,11 +2,54 @@ import Duck from '@/components/Duck';
 import FancyRectangle from '@/components/FancyRectangle';
 import ImageCarousel from '@/components/ImageCarousel';
 import Title from '@/components/Title';
+import { EVENTS, type Event } from '@/data/events';
 import { CAROUSEL_IMAGES } from '@/data/home';
 import { SPONSOR_TYPES, getSponsors } from '@/data/sponsors';
 import Image from 'next/image';
 import Link from 'next/link';
 import { Fragment } from 'react';
+import { FiClock, FiMapPin } from 'react-icons/fi';
+
+function UpcomingEventCard({ event, index }: { event: Event; index: number }) {
+    return (
+        <FancyRectangle colour="white" offset="8" rounded fullWidth>
+            <div className="flex w-full flex-col gap-6 rounded-xl bg-white p-4 text-black lg:flex-row">
+                <div className="grow space-y-2 md:space-y-4">
+                    <div className="flex gap-6">
+                        <div
+                            className={`h-fit rounded-md border-[3px] border-black px-4 py-2 text-xl ${['bg-orange', 'bg-yellow', 'bg-purple'][index % 3]}`}
+                        >
+                            <div>{event.date.month}</div>
+                            <div>{event.date.day}</div>
+                        </div>
+                        <div className="grow space-y-2 text-2xl">
+                            <h4 className=" md:border-b-[3px] md:border-black md:pb-1 ">
+                                {event.title}
+                            </h4>
+                            <div className="flex gap-2">
+                                <FiClock size={26} />
+                                <span>{event.time}</span>
+                            </div>
+                            <div className="flex gap-2">
+                                <FiMapPin size={26} />
+                                <span>{event.location}</span>
+                            </div>
```

Comment:
Make the event time and location text smaller and not bold by adding `text-lg font-normal` to the `className`. Doing this should help fix the icons being different sizes.